packages/connector-wechat-native/src/index.ts

```diff
@@ -68,17 +65,16 @@ export default class WeChatNativeConnector implements SocialConnector {
       access_token: accessToken,
       openid,
       errcode,
+      errmsg,
     } = await got
       .get(accessTokenEndpoint, {
         searchParams: { appid, secret, code, grant_type: 'authorization_code' },
         timeout: defaultTimeout,
       })
       .json<AccessTokenResponse>();
 
-    assert(
-      errcode !== 40_029 && accessToken && openid,
-      new ConnectorError(ConnectorErrorCodes.SocialAuthCodeInvalid)
-    );
+    assert(errcode !== 40_029, new ConnectorError(ConnectorErrorCodes.SocialAuthCodeInvalid));
+    assert(!errcode && accessToken && openid, new Error(errmsg));
```

Comment:
just curious: better use errcode when errmsg is blank?

should align with below `errmsg ?? ''`